File: examples/go/currency/rates/getRatesForUsd.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/currency"
)

// Rates returns the currency rates for a given code e.g USD
func GetRatesForUsd() {
	currencyService := currency.NewCurrencyService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := currencyService.Rates(&currency.RatesRequest{
		Code: "USD",
	})
	fmt.Println(rsp, err)
}

File: examples/js/currency/rates/getRatesForUsd.js
const { CurrencyService } = require("m3o/currency");

// Rates returns the currency rates for a given code e.g USD
async function getRatesForUsd() {
  let currencyService = new CurrencyService(process.env.MICRO_API_TOKEN);
  let rsp = await currencyService.rates({
    code: "USD",
  });
  console.log(rsp);
}

getRatesForUsd();
